src/CattyTests/DataModel/VariablesContainerTest.swift

```diff
@@ -459,6 +459,30 @@ final class VariablesContainerTest: XCTestCase {
         XCTAssertFalse(container.isProjectVariableOrList(list2))
     }
 
+    func testIsProjectVariable() {
+        let userVariable1 = UserVariable(name: "testVar1", isList: false)
+        let userVariable2 = UserVariable(name: "testVar2", isList: false)
```

Comment:
Please add `userVariable2` as an object variable for a object to distinguish between object and program variables (that's the purpose of the function you are testing).